toolchain/semantics/semantics_parse_tree_handler.cpp

```diff
@@ -130,7 +130,7 @@ auto SemanticsParseTreeHandler::Push(ParseTree::Node parse_node) -> void {
                 << parse_tree_->node_kind(parse_node) << "\n";
   CARBON_CHECK(node_stack_.size() < (1 << 20))
       << "Excessive stack size: likely infinite loop";
-  node_stack_.push_back({parse_node, std::nullopt});
+  node_stack_.push_back({parse_node, SemanticsNodeId()});
```

Comment:
I think it would be helpful to add a static `Invalid` factory function to `SemanticsNodeId` so you can write` SemanticsNodeId::Invalid()` here.

Reply:
Done (MakeInvalid for consistency)